v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -102,7 +90,7 @@
     "isMobile": true
   },
   {
-    "destKey": "$user_agent",
+    "destKey": "$useragent",
```

Comment:
Is this correct? Are we seeing correct values in destination?

Reply:
Is this correct?  -> yes
Are we seeing correct values in destination? -> yes